packages/form/src/hooks/use-field.js

```diff
@@ -42,7 +42,10 @@ export const useField = ( { name: fieldName, type, value } ) => {
 	const inputProps = {
 		name: fieldName,
 		onChange,
-		value: type === 'checkbox' || type === 'radio' ? value : currentValue,
+		value:
+			type === 'checkbox' || type === 'radio'
+				? value
+				: currentValue || '',
```

Comment:
Added this to avoid React's uncontrolled to controlled error on the input fields. Is there a better way to handle this?

Reply:
I guess this is about `currentValue` initially being `null` or `undefined` until you start editing it?
Like yourself, I'm not sure if we might be missing something here but that would only become obvious once we start adding more advanced forms. So for the time being, I think this is fine.

Reply:
> I guess this is about currentValue initially being null or undefined until you start editing it?
Yes, this is the case. Let's keep it this way for now then.